### ISSUE TITLE
Discussion of session 6 readings: Seifried 2019 + Turchetto 2017

### ISSUE THREAD

**Issue by gabrielbodard** [SunoikisisDC/SunoikisisDC-2020-2021#29]
* Seifried, R.M. and C.A.M. Gardner. 2019. "Reconstructing historical journeys with least-cost analysis: Colonel William Leake in the Mani Peninsula, Greece." _Journal of Archaeological Science: Reports_ 24:391-411. Pre-print available: https://works.bepress.com/rebecca-seifried/11/
* Turchetto, J. and G. Salemi. 2017. “Hide and Seek. Roads, Lookouts and Directional Visibility Cones in Central Anatolia.” _Open Archaeology_ 3:69-82. Available: https://doi.org/10.1515/opar-2017-0004

Please consider questions of provenance, context, authorship and venue, relevance to the discussion of GIS in particular and geo-technologies in general, and the rest of this semester's content.

**Comment by nicolealexandra33** [comment]
**Truchetto and Salemi**
This article investigates Roman routes in Central Turkey (Cappadocia) that were reused in the seventh century AD by the Byzantine/Arab military. Scholars attempt to locate the exact routes by means of both written sources and archaeological evidence: ‘In  particular,  this  is  the  case  of  the  road  network  established  between  Koloneia/Aksarayand  Tyana/Kemerhisar, running both along the feet and across the volcanic group formed by Hasan Dağı, Keçiboydoran Dağı, Melendiz Dağları and Göllü Dağı’. During Roman times, this route was considered a ‘secondary’ route as it passed through the mountains, and ancient sources describe more direct routes in the region that went around the mountain range. They explain that part of the reason for the change in route popularity between Roman and Byzantine times was because of the region’s increase of being raided by Arab forces which forced a social ‘fragmentation’, as well as Cappadocia became reinforced by Byzantine military forces as it was the border of the Byzantine Empire and the surrounding forces.
	
Their research questions: ‘To what extent did the central Cappadocian fortresses match the picture recalled by the written sources? To what extent did they face the needs derived from the new offensive/defensive strategy? What kind of control did they exercise over that frontier territory, and which was the relationship established between the fortresses and the routes crossing the volcanic district of Melendiz Dağları, Hasan Dağı and Göllü Dağı?’
	
Using GIS technology, they were able to determine the views from their estimated fortress locations (based on ancient sources and where archaeological finds were located). From this viewpoints, they could also see how much of these routes were visible from the fortress. In turn, this could also help to determine the relationship between fortresses and how well they were connected. Finally, they studied viewpoints from the fortresses in all directions, in order to ascertain which orientation would have provided the best control and visibility for defence.
	
Using GIS, they were able to find out the location, orientation, and viewpoints of these fortresses which they determined were placed, in order to have the greatest visibility of the surrounding area and of the mountain routes, ‘especially  across  Hasan  Dağı,  Keçiboydoran  Dağı,  Melendiz  Dağları  and  Göllü  Dağı,  and, to a lesser extent, along the southern slopes of those same mountains. In particular, three out of the four routes linking Koloneia and Tyana, even if to varying degrees, fell into the 4 km-radius of the visibility cones generated from the fortresses we have considered’. By having a decent viewpoint of these routes, the military (in the fortresses) were also able to monitor entry points into the region by enemies, as well as able to communicate efficiently along these routes to other fortresses. However, from their analysis, it seems as though the fortress network had one weakness and that was their lack of visibility to the southeast over the volcanic area. They conclude, ‘At the time of the Arab raids an efficient road network should have been suitable for quick and rapid escapes to safer and better protected places, equipped with defensive structures such as forts and fortresses, but still linked to the main axes of communication traversing Cappadocia and, thus, making that border territory an effective ‘central periphery’ of the Byzantine Empire’.

- Jacopo Turchetto, did his PhD in landscape archaeology, but is currently within the department of ‘Cultural Heritage: Archaeology and History of Art, Cinema and Music’ at the University of Padova
- Giuseppe Salemi is an associated professor at the University of Padova, also within the department of ‘Cultural Heritage: Archaeology and History of Art, Cinema and Music’
- Given the relevance of geographical features, such as mountain ranges and volcanoes, Turchetto’s training in landscape archaeology would have been particularly applicable to this project

**Comment by despinaborcea** [comment]
Seifried and Gardner’s article discusses the assessment of the GIS least-cost analysis tool, which uses least-cost paths (LCPs) to model movement across landscapes and remains one of the most common GIS tools in use by archaeologists. The authors are mindful of two extreme trends with the tool: while it is a user-friendly instrument, least-cost analysis does not always deliver the most accurate results. The alternative, designing and implementing more complicated algorithms to model movement across landscapes, could render a more accurate outcome, but it might be less accessible due to the incurring costs of creating the algorithms, as well as the users’ computational understanding and experience. In this context, Seifried and Gardner attempt to find a middle-ground by using Colonel William Martin Leake’s thoroughly documented account of his visit to the Mani Peninsula in Greece in 1805 to compare it with least-cost analyses in GIS and assess its accuracy levels. 

Section 3 discusses the case study of the Mani Peninsula, which is a particularly relevant example because of the region’s high preservation status. A brief discussion on the history of the Peninsula follows, from the ancient coastal settlements mentioned by Pausanias in his Periegesis to the increased agricultural activity and population during the medieval period. In 1453, the Ottoman Empire conquered Mani, which became an important focal point for the diplomatic affairs of Mediterranean states until the Greek Revolution in 1821. During this interval, frictions between the Greeks and Ottomans were reflected in the local landscape, marked by the construction of both Ottoman fortresses and footpaths built by the locals. From the mid-20th century onwards, the population of Mani has decreased due to the urban migration of the locals, which contributed to the preservation of the local footpaths and the kalderimia (stone-paved roads primarily built for hoofed animals transport). The final part of the section summarises key points about the trajectory of Colonel Leake’s travels across Mani.

Methodology is discussed in Section 4. As the aim of the article is to discuss results obtained from testing various least-cost methods for movement modelling across past landscapes using historical waypoints (sets of coordinates in GIS identifying a physical point), least-cost analysis renders relevant. The authors test 8 methods for calculating LCPs between Leake’s journey waypoints: except for one based on slopes and another on the presence/absence of pre-existing path, all the remaining 6 were based on different time-functions. The steps are as following:

- defining a cost raster: determining parameters which will disrupt movement across a trajectory and assigning these parameters numerical values (for example, a ‘no-cost’ raster for a time-based analysis defines the shortest path from point A to point B)
- calculating a total accumulated cost surface and backlink raster for each waypoint: the ACS encapsulates a movement over a landscape and every alternative trajectory has a particular cost (depending on the chosen rasters, some will cost more than others)
- calculating a LCP between each set of waypoints: using the ACS data, the authors calculate the least-cost paths for Leake’s journey according to different rasters (e.g., time/slope)
- calculating a low-cost corridor: to assess the results, the authors consider comparing Leake’s route with the GIS least-cost analysis undertaken so far. However, as they do not have an exact description of his journey, but only references to landmarks on his way, they create low-cost corridors.

Following the comparison with Leake’s account, the authors note that most of his routes match those predicted by LCPs and that one important ‘cost’ in his trip was slope. On one occasion, however, his divergence from the least-cost path was not predicted by any of the 8 methods, because he deviated from the most efficient routes for a better view. 

In conclusion, the authors propose the application of GIS least-cost analysis tool for other case studies, having rendered it useful in predicting least-cost effective paths for travelling in the Mani Peninsula after a comparison with Leake’s account. They appreciate that cost rasters cannot reflect all ‘cost’ factors: for example, due to rising frictions between the English and the locals from Pyrgos Dirou, Leake and his companions were forced to deviate to a different, less effective route than the predicted LCPs. However, it is precisely deviations like this from LCPs calculations that may illuminate other non-factorable ‘costs’ in a route (e.g. here, political grounds).

**Comment by K-Doering** [comment]
>...In this context, Seifried and Gardner attempt to find a middle-ground by using Colonel William Martin Leake’s thoroughly documented account of his visit to the Mani Peninsula in Greece in 1805 to compare it with least-cost analyses in GIS and assess its accuracy levels.
> 
> In conclusion, the authors propose the application of GIS least-cost analysis tool for other case studies, having rendered it useful in predicting least-cost effective paths for travelling in the Mani Peninsula after a comparison with Leake’s account. They appreciate that cost rasters cannot reflect all ‘cost’ factors: for example, due to rising frictions between the English and the locals from Pyrgos Dirou, Leake and his companions were forced to deviate to a different, less effective route than the predicted LCPs. However, it is precisely deviations like this from LCPs calculations that may illuminate other non-factorable ‘costs’ in a route (e.g. here, political grounds).
>

Thanks to both for your great summaries! I was particularly interested in the marriage of historical narrative and the GIS tools. As is a common theme in the previous tools and projects we have seen (and as I am typing is quite obviously inherent in 'digital humanities'), I think that these are invaluable methods of analysis for quantifying narrative experience to gain new meaning and understanding. It was a great note by the authors that the aim of the narratives is important to take into consideration. Narratives can be categorized such as the work of Leake, which reinforces our ability to trust his precision since the aim was militarial. 

I think it would also be useful to interview locals and ask how their families have traveled the area in the recent past (while we still have access to knowledge of a time before widespread use of automobiles in our family histories). I do think it is quite true that once we have "hacked" a human problem we tend to hold tight to those ways of doing things until we find a new, better solution. But, of course, there are exceptions such as the most 'logical' route did not give Leake the best survey of the environment so we must look at the situations on a case-by-case basis when doing analysis.

**Comment by nicolealexandra33** [comment]
I was trying to see if maybe either of the applications of GIS that are mentioned in the articles (viewshed and LCPs) were used to try and clarify any of the routes taken by the Franklin expedition in the Arctic Circle once they went 'missing' from the records, but besides LIDAR, I am not sure they have used these applications, or how useful they would be. Viewshed might not be the most helpful for the Arctic, especially since the expedition wasn't setting up settlement for defence, but maybe LCPs would be useful? There would be other factors to assess as well, such as seasonal changes as well, but maybe this along with LCPs might illuminate some of their potential trajectories

**Comment by despinaborcea** [comment]
> > ...In this context, Seifried and Gardner attempt to find a middle-ground by using Colonel William Martin Leake’s thoroughly documented account of his visit to the Mani Peninsula in Greece in 1805 to compare it with least-cost analyses in GIS and assess its accuracy levels.
> > In conclusion, the authors propose the application of GIS least-cost analysis tool for other case studies, having rendered it useful in predicting least-cost effective paths for travelling in the Mani Peninsula after a comparison with Leake’s account. They appreciate that cost rasters cannot reflect all ‘cost’ factors: for example, due to rising frictions between the English and the locals from Pyrgos Dirou, Leake and his companions were forced to deviate to a different, less effective route than the predicted LCPs. However, it is precisely deviations like this from LCPs calculations that may illuminate other non-factorable ‘costs’ in a route (e.g. here, political grounds).
> 
> Thanks to both for your great summaries! I was particularly interested in the marriage of historical narrative and the GIS tools. As is a common theme in the previous tools and projects we have seen (and as I am typing is quite obviously inherent in 'digital humanities'), I think that these are invaluable methods of analysis for quantifying narrative experience to gain new meaning and understanding. It was a great note by the authors that the aim of the narratives is important to take into consideration. Narratives can be categorized such as the work of Leake, which reinforces our ability to trust his precision since the aim was militarial.
> 
> I think it would also be useful to interview locals and ask how their families have traveled the area in the recent past (while we still have access to knowledge of a time before widespread use of automobiles in our family histories). I do think it is quite true that once we have "hacked" a human problem we tend to hold tight to those ways of doing things until we find a new, better solution. But, of course, there are exceptions such as the most 'logical' route did not give Leake the best survey of the environment so we must look at the situations on a case-by-case basis when doing analysis.

I certainly agree, Keyah - the most logical predicted trajectory is not necessarily the most feasible in real life. I found it quite fascinating how LCPs are important for both the reconstruction stories they tell and for the ones they do not - in Leake's case, his unpredicted diversions had historical motivations behind them, but I think one important aspect is that we know those motivations (and this is one of the reasons the authors chose his example, together with the fact that his trip was well-documented). I can see complications in reconstructive scenarios if the trajectory of a person/people is not predicted by LCPs and we cannot justify the reason behind the detour at first sight. This is why I think that, far from a shortcoming, when held against real-life and unpredicted deviations, LCP analyses may enable interesting and potentially important questions about the reasons behind those route discrepancies.

**Comment by FabioDFernandes** [comment]
Thank you Nicole and Despina for your summaries! In the case of the Cappadocia fortresses, it does seem logically obvious that fortresses would be placed where they could accumulate a view over the surrounding territory and passes from where outsiders could enter, as well as of other fortresses for communication, but it is certainly useful to have GIS validate that with much more precise and tangible data that the eye and logic alone cannot provide, and thus also revealing any shortcomings that people at the time may not have accounted for (i.e. their lack of visibility to the southeast over the volcanic area). 

In the case of the Mani project, it is fascinating to see how we can further reconstruct voyages from the past and so, in the wider application of this, come to understand so much more - perhaps, patterns of migration, trade and carrying of material/cargoes, general travel, and how long this might have taken, and hence we could deduce how this might have impacted things locally on an economic and societal level (although maybe I am overthinking this). The discussion above has done well to point out the limitations that abound here, though even if there are so many missing details from past journeys that we do not possess, LCP can still illuminate a lot and help us reconstruct on a more 'logical' basis. I think @K-Doering 's idea about interviewing locals in the region is a really neat one, especially as this knowledge is bound to disappear with the rural-urban emigration typical in Greece and elsewhere, and the use of vehicles and construction of roads.

**Comment by ChantalvanEgdom** [comment]
Thank you both for the great summaries! I really like the idea of challenging written sources in this way, and as @FabioDFernandes has pointed out, it is a great way of revealing shortcomings. Also good to see how helpful GIS tools can be in establishing such arguments.

It'd be interesting to look at tensions between results from cost rasters and actual, real life decision making and see whether that is due to said limitations in human abilities, or if other reasons could be found (dangerous routes for example). @K-Doering 's point of using local knowledge would be beneficial to that, but when looking at much earlier times it would be great if sources could confirm the idea of "dangerous routes" due to things like robberies or lesser quality roads.

**Comment by RebeccaKimberlin** [comment]
> Thank you both for the great summaries! I really like the idea of challenging written sources in this way, and as @FabioDFernandes has pointed out, it is a great way of revealing shortcomings. Also good to see how helpful GIS tools can be in establishing such arguments.
> 
> It'd be interesting to look at tensions between results from cost rasters and actual, real life decision making and see whether that is due to said limitations in human abilities, or if other reasons could be found (dangerous routes for example). @K-Doering 's point of using local knowledge would be beneficial to that, but when looking at much earlier times it would be great if sources could confirm the idea of "dangerous routes" due to things like robberies or lesser quality roads.

This is such a good point, re tensions between the results. Especially when routes continue to be used, but certain threats may have faded, it emphasises how the technology and written sources work and thrive off each other, filling gaps that the other has - and, as Despina highlighted above, the LCPs cannot account for the human decision making, to avoid certain routes because of social/political factors rather than geography/lower cost. I found it really interesting too, for example, that Leake’s group diverted through a village so that someone could collect their musket before continuing, which could not be explained otherwise, but mapped within the rest of his account, offers a fuller picture of the journey and its aims.

**Comment by rmseifried** [comment]
Hi all, I just wanted to chime in and respond to a couple of points that folks raised. @RebeccaKimberlin brought up the idea to take social/political factors into account when running LCA, and it made me think of this section ([5.7 Social and cultural cost components](https://intarch.ac.uk/journal/issue36/5/5-7.html)) of Irmela Herzog's larger review article on archaeological LCA studies. It is not an insurmountable challenge, but not one that many have taken on!

I also love @K-Doering's suggestion about interviewing locals about the ways they move across the landscape. When I was doing fieldwork to map the pre-modern path network (which used in the LCA analysis in the article you read), several of the older family members I was staying were the main sources of information about where they were located. One pointed out the exact route she used to take to walk to school every day - a path 6km long that is now mostly overgrown. We haven't done interviews like that elsewhere in the region, and that is an excellent idea!

**Comment by FabioDFernandes** [comment]
@rmseifried Thank you for the interesting section on this matter - I would imagine, especially in the study of past routes, it is certainly a challenge. And how interesting to learn about your experience with older family members. Especially in non-urbanised, but populated, settings with heavily restrictive geography (like the rugged Mani Peninsula), I would think it logical that certain paths have been in use for centuries, even millennia, due to their conduciveness in getting around, unless certain social/political factors were to interfere.

**Comment by Kiamanx** [comment]
Absolutely fascinating. It is interesting to think about the sheer number of ancient routes and desire paths that have been eroded with time. About a year ago, I was looking into various routes in and out of the southern Andalucian mountains between the ocean and the plains. There were hundreds of different routes that still remained with ancient villas and sites marking where some lay such as the "Route of the Caliphs" (which I believe is what the modern highways are situated upon) - but a vast majority of the routes have been lost to time. I would love to see a detailed study or a GIS overview on that region since so many nations have had to find their own routes through the mountainous passes. 

I definitely agree with Fabio's point that the futute implications of GIS would lead to the uncovering of some of the more ancient routes. It would also be interesting to see if the LCP is the one that is used for the modern road.

**Comment by PAZHames** [comment]
@K-Doering you've started an inter-generational revolution of data mining! I also love how this ties into what we were thinking about last week re the importance of harvesting a generation's knowledge on an increasingly changing/disappearing political landscape like Palestine.

**Comment by alexandrabushman** [comment]
> > > ...In this context, Seifried and Gardner attempt to find a middle-ground by using Colonel William Martin Leake’s thoroughly documented account of his visit to the Mani Peninsula in Greece in 1805 to compare it with least-cost analyses in GIS and assess its accuracy levels.
> > > In conclusion, the authors propose the application of GIS least-cost analysis tool for other case studies, having rendered it useful in predicting least-cost effective paths for travelling in the Mani Peninsula after a comparison with Leake’s account. They appreciate that cost rasters cannot reflect all ‘cost’ factors: for example, due to rising frictions between the English and the locals from Pyrgos Dirou, Leake and his companions were forced to deviate to a different, less effective route than the predicted LCPs. However, it is precisely deviations like this from LCPs calculations that may illuminate other non-factorable ‘costs’ in a route (e.g. here, political grounds).
> > 
> > 
> > Thanks to both for your great summaries! I was particularly interested in the marriage of historical narrative and the GIS tools. As is a common theme in the previous tools and projects we have seen (and as I am typing is quite obviously inherent in 'digital humanities'), I think that these are invaluable methods of analysis for quantifying narrative experience to gain new meaning and understanding. It was a great note by the authors that the aim of the narratives is important to take into consideration. Narratives can be categorized such as the work of Leake, which reinforces our ability to trust his precision since the aim was militarial.
> > I think it would also be useful to interview locals and ask how their families have traveled the area in the recent past (while we still have access to knowledge of a time before widespread use of automobiles in our family histories). I do think it is quite true that once we have "hacked" a human problem we tend to hold tight to those ways of doing things until we find a new, better solution. But, of course, there are exceptions such as the most 'logical' route did not give Leake the best survey of the environment so we must look at the situations on a case-by-case basis when doing analysis.
> 
> I certainly agree, Keyah - the most logical predicted trajectory is not necessarily the most feasible in real life. I found it quite fascinating how LCPs are important for both the reconstruction stories they tell and for the ones they do not - in Leake's case, his unpredicted diversions had historical motivations behind them, but I think one important aspect is that we know those motivations (and this is one of the reasons the authors chose his example, together with the fact that his trip was well-documented). I can see complications in reconstructive scenarios if the trajectory of a person/people is not predicted by LCPs and we cannot justify the reason behind the detour at first sight. This is why I think that, far from a shortcoming, when held against real-life and unpredicted deviations, LCP analyses may enable interesting and potentially important questions about the reasons behind those route discrepancies.

Thank you both for your summaries! That's a great point, detours could have occurred for a myriad of reasons that digital cultures cannot convey. @ChantalvanEgdom made a great point, we could probably fill these gaps to identify dangerous routes with primary sources.
